services/frontend/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "gatsby-starter-hello-world",
+  "private": true,
+  "description": "A simplified bare-bones starter for Gatsby",
+  "version": "0.1.0",
+  "license": "0BSD",
+  "scripts": {
+    "build": "gatsby build",
```

Comment:
Can we add a README.md to explain what these do?

Reply:
I'll type one up now, but most of them or for internal use in gatsby.